pkg/install/v1alpha1/cstor_pool.go

```diff
@@ -186,8 +182,6 @@ spec:
         app: cstor-pool
         openebs.io/version: {{ .CAST.version }}
         openebs.io/cas-template-name: {{ .CAST.castName }}
-      annotations:
-        openebs.io/monitoring: pool_exporter_prometheus
       ownerReferences:
```

Comment:
this `ownerReferences` also seems to be part of pool-exporter runTask PR? Do we still need this?

Reply:
@vishnuitta just verified this in PR, this was there earlier also
https://github.com/openebs/maya/pull/960/files